composer.json

```diff
@@ -224,7 +224,10 @@
                 "3417771 : Some styling no longer applies due to form-actions -> field-actions html change (https://git.drupalcode.org/project/paragraphs/-/merge_requests/93.diff)": "patches/drupal/paragraphs/3417771.diff"
             },
             "drupal/purge": {
-                "3094343 : Garbled purge tag IDs": "https://www.drupal.org/files/issues/2023-09-07/purge-3094343-15.patch"
+                "3094343 : Garbled purge tag IDs": "https://www.drupal.org/files/issues/2023-09-07/purge-3094343-15.patch",
+                "3460094 : There is no purger supporting array":  "patches/drupal/purge/3460094.diff",
+                "3460094 : Remove drupal.services.yml":  "patches/drupal/purge/3460094-2.diff"
```

Comment:
This needs a link to where the patch came from.  See #4477 (issue #4424) for examples.